optuna/pruners/hyperband.py

```diff
@@ -59,6 +64,10 @@ class HyperbandPruner(BasePruner):
             :math:`\\eta` in the paper. See the details for
             :class:`~optuna.pruners.SuccessiveHalvingPruner`.
         n_brackets:
+
+            .. deprecated:: 2.0.0
```

Comment:
This deprecated section will be rendered as follows:

> Deprecated since version 2.0.0: This argument will be removed from class:~optuna.pruners.HyperbandPruner.

(See https://37099-122299416-gh.circle-artifacts.com/0/docs/build/html/reference/pruners.html#optuna.pruners.HyperbandPruner)

Could you tell me why you choose `2.0.0` instead of `1.4.0`?

```suggestion
            .. deprecated:: 1.4.0
```

Reply:
I thought this change is distractive for the `HyperbandPruner`, so the change should be done in a major release. Could you tell me how we deal with this type of distractive change?

Reply:
I fixed `2.0.0` to `1.4.0`.

Reply:
Sorry for the delayed response, but you have made the change to keep the backward compatibility.

>  Could you tell me how we deal with this type of distractive change?

The `HyperbandPruner` is an experimental feature and I think we do not ensure the compatibility between minor releases.